Intel_SighPP_CMake/Intel_SighPP_CMake/src/Intel_Sight_++.cpp

```diff
@@ -192,7 +200,33 @@ int main(int argc, char** argv)
 			{
 				SPDLOG_ERROR("Missing flag/argument for loading the darknet-based yolo network");
 				continue;
+			}	
+
+			if(next_arg.compare("-outdoors") == 0)
+			{
+				theme = 1;
+				for(auto i = outdoors_model.begin(); i != outdoors_model.end(); ++i){
+					std::pair<std::set<string>::iterator,bool> ret = caffe_models_names.insert(*i);
+					if(ret.second){
+						CaffeModelImpl caffe_model = create_caffe_network(*i);
+						caffe_model.set_resolution(300,300);
+						caffe_models.push_back(caffe_model);
+						SPDLOG_INFO("Caffe-based network loaded for outdoors environment:  {}", *i);
+					}
+				}
 			}
+			else if(next_arg.compare("-indoors") == 0)
+			{
+				theme = 2;
+				for(auto i = indoors_model.begin(); i != indoors_model.end(); ++i){
+					std::pair<std::set<string>::iterator,bool> ret = caffe_models_names.insert(*i);
+					if(ret.second){
+						CaffeModelImpl caffe_model = create_caffe_network(*i);
+						caffe_model.set_resolution(640,480);
```

Comment:
Can't this be set in the constructor?

Reply:
Also remember that the realsense camera default setup is 640x480 if that matters for this purpose.

Reply:
Yeah we can set it in the constructor but I think it is also acceptable to have a separate function. So in some cases you may need to use the same network but with different resolution so by using a separate function you don't have to initiate another one.